src/conduit/components/forms.cljs

```diff
@@ -22,17 +22,21 @@
              v))])])
 
 (rum/defc InputField
-  [{:keys [placeholder type value errors on-blur on-focus on-change]}]
-  [:fieldset.form-group
-   [:input.form-control.form-control-lg
-    {:placeholder placeholder
-     :on-change #(on-change (.. % -target -value))
-     :on-blur on-blur
-     :on-focus on-focus
-     :type (or type :text)
-     :value value}]
-   (when errors
-     (InputErrors errors))])
+  [{:keys [placeholder type value errors on-blur on-focus on-change field-type]}]
+  (let [input-field (case field-type
+                :textarea (fn [attrs & children]
+                            (apply vector :textarea.form-control.form-control-lg (assoc attrs :rows 8) children))
```

Comment:
Do not construct elements programatically, it causes them to be interpreted at runtime which leads to perf issues, also pure Hiccup is much more readable. This should be defined as a static Hiccup element.